patches/chromium/desktop_media_list.patch

```diff
@@ -153,3 +153,12 @@ index b2005c70acbc1c05c59bb2059b190ab78fb63a68..1a7188e2df76672d66da3206d4448df3
  #include "media/base/video_util.h"
  #include "third_party/libyuv/include/libyuv/scale_argb.h"
  #include "third_party/skia/include/core/SkBitmap.h"
+@@ -218,6 +219,8 @@ void NativeDesktopMediaList::Worker::RefreshThumbnails(
+       FROM_HERE, {BrowserThread::UI},
+       base::BindOnce(&NativeDesktopMediaList::UpdateNativeThumbnailsFinished,
+                      media_list_));
++
++  capturer_.reset();
```

Comment:
It looks like there is a `PostTask` just above this that does work, are we sure resetting here doesn't interfere with that task?

Reply:
i don't believe so - it's the exact same flow as occurred [here](https://github.com/electron/electron/pull/14835/files#diff-22735fa1b90e850a4ac270613eb16dcbL214-L215) with the same PostTask